numpy/typing/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 .. versionadded:: 1.20
 
-Large parts of the NumPy API have PEP-484-style type annotations. In
+Large parts of the NumPy API have PEP-484-style[https://peps.python.org/pep-0484/] type annotations. In
```

Comment:
```suggestion
Large parts of the NumPy API have :pep:`484`-style type annotations. In
```
You can directly link to the relevant pep using Sphinx's `:pep:` role.